boilerplate/app/screens/DemoShowroomScreen/DemoShowroomScreen.tsx

```diff
@@ -97,7 +97,7 @@ export const DemoShowroomScreen: FC<DemoTabScreenProps<"DemoShowroom">> =
 
     // handle Web links
     React.useEffect(() => {
-      if (route.params) {
+      if (Object.keys(route.params).length > 0) {
```

Comment:
let's add a `Platform.OS` check for web here because it still remains open on mobile from what I tested.

Reply:
It appears to work as intended in mobile for me, can you guide me to how you tested ? i'm relatively new to using expo and react-native

Reply:
@ludocourbin I was testing with `npx ignite-cli@next`, let me try it again with the master version and if that works I can make the necessary updates for next as a follow-up

Reply:
Perfect, thank you